src/Constants/commands_messages.py

```diff
@@ -1,2 +1,14 @@
-CREATE_PROFILE_NAME = "What would you like to name your profile?: "
-CREATE_PROFILE_SUCCESS_MESSAGE = "Successfully created profile. Now switching to profile: {}"
\ No newline at end of file
+CANCEL_ACTION_MESSAGE = "Canceling action: {}"
+CREATE_FREEZER_ACTION = "Create freezer with name '{}'."
+CREATE_FREEZER_NAME_MESSAGE = "What would like to name this freezer?: "
+CREATE_FREEZER_OVERRIDE_NAME = "A freezer with the name '{}' already exists. " + \
```

Comment:
Yeah I thought it would be good to allow a profile to add multiple fridges/freezers because it doesn't require much more code changes on our end I guess. I was thinking this could be useful if they had like a mini fridge in one room or multiple fridges.